src/renderer/screens/exchange/Swap2/History/HistoryLoading.js

```diff
@@ -0,0 +1,12 @@
+// @flow
+import React from "react";
+import Box from "~/renderer/components/Box/Box";
+import BigSpinner from "~/renderer/components/BigSpinner";
+
+const HistoryLoading = () => (
+  <Box flex={1} alignItems="center" justifyContent="center">
+    <BigSpinner size={50} />
```

Comment:
I think the loading spinner for providers is larger than this? they should be the same, whatever figma says

Reply:
Got it!

Reply:
Change here [3d35d310bd344ac1099ff4e873ca696371e0ad5c](https://github.com/juan-cortes/ledger-live-desktop/pull/14/commits/3d35d310bd344ac1099ff4e873ca696371e0ad5c)